### DISCUSSION TITLE
How to bind rocket to multiple network interfaces?

### DISCUSSION THREAD

**Discussion by chabapok** [rwf2/Rocket#2231] [Questions]
I want to run rocket-based app on multiple network interfaces and/or ports. How to do this?

Configure rocket to bind to `0.0.0.0` does not suit me. I want to specify a specific list of addr:port

**Comment by edgarogh** [comment]
Rocket can only listen on one IP address[^1] for now (not even Unix sockets [^2]). You might want to play with your OS' tools, like `iptables` on Linux to achieve what you want to do. I have no idea how it works so I cannot help you more, but I'm pretty sure that there is a way to do this at OS level. Docker can make crazy things related to networks, I'm sure you can too.

[^1]: https://docs.rs/rocket/0.5.0-rc.2/rocket/config/struct.Config.html#structfield.address
[^2]: #545
